docs/user_guide/Upgrade_SIMP/General_Upgrade_Instructions.rst

```diff
@@ -180,14 +113,16 @@ Incrementally upgrading systems using local repositories
 
       puppet agent -t
 
+Other deployment scenario incremental upgrade
```

Comment:
This topic title is less meaningful than the previous version.

* ["Upgrading" made it clearer](http://www.informit.com/articles/article.aspx?p=327993&seqNum=5) that this section describes a task.
* "Other deployment scenario [sic]" is less useful to readers, and less accurate:  As more scenarios are added (e.g., bootstrapping via `bolt`), they won't necessarily involve r10k, Code Manager, or even Control Repositories.

The common feature between using r10k or using Code Manager is that both deploy from a control repository.  Maybe this topic would be better as **"Incrementally upgrading systems that use control repositories"**?

Reply:
I am not sure what you are saying is less clear.  I added back in the Example for a CR and the link works now.

Reply:
> I am not sure what you are saying is less clear. I added back in the Example for a CR and the link works now.

I'm just talking the wording of this heading: _"Incrementally upgrading X"_  was clearer than _"X incremental upgrade"_.  The "-ing" (gerund) form of the verb made it easy to tell that this section details a _task_ (as opposed to a describing a concept or reference information).

* http://www.informit.com/articles/article.aspx?p=327993&seqNum=5
* http://stylepedia.net/style/#heading-styles

I'll submit a patch to show what I mean.